pages/vi/vi-team.md

```diff
@@ -46,7 +46,7 @@
 |[gowiredu](profiles/gowiredu.md)|2018-06-22|
 |[NestorSaavedra](profiles/saavfoxdev.md)|2018-06-21|
 |[rkinback](profiles/rkinback.md)|2018-07-02|
-
+|[leahphuong](profiles/leahphuong.md)|2018-07-10|
```

Comment:
@leahphuong Could make a change to keep the empty line between the list and the next header (`## Former Interns`)?

Reply:
@paulbert thanks ;)

Reply:
@paulbert done!